### PR TITLE
Make parser error messages more explicit

### PR DESCRIPTION
**Pull request by hoodie**

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -256,7 +256,7 @@ pub fn date(string: &str) -> Result<Date, String> {
     if let Ok((_, parsed)) = parsers::parse_date(string.as_bytes()) {
         Ok(parsed)
     } else {
-        Err(format!("Parser Error: {}", string))
+        Err(format!("Failed to parser Date: {}", string))
```
Comment (badboy) [inline src/lib.rs:259]:
```suggestion
        Err(format!("Failed to parse date: {}", string))
```

File: src/lib.rs [hunk 2]
```diff
@@ -278,7 +278,7 @@ pub fn time(string: &str) -> Result<Time, String> {
     if let Ok((_, parsed)) = parsers::parse_time(string.as_bytes()) {
         Ok(parsed)
     } else {
-        Err(format!("Parser Error: {}", string))
+        Err(format!("Failed to parser Time: {}", string))
```
Comment (badboy) [inline src/lib.rs:281]:
```suggestion
        Err(format!("Failed to parse time: {}", string))
```

File: src/lib.rs [hunk 3]
```diff
@@ -297,7 +297,7 @@ pub fn datetime(string: &str) -> Result<DateTime, String> {
     if let Ok((_left_overs, parsed)) = parsers::parse_datetime(string.as_bytes()) {
         Ok(parsed)
     } else {
-        Err(format!("Parser Error: {}", string))
+        Err(format!("Failed to parser Datetime: {}", string))
```
Comment (badboy) [inline src/lib.rs:300]:
```suggestion
        Err(format!("Failed to parse datetime: {}", string))
```

File: src/lib.rs [hunk 4]
```diff
@@ -336,6 +336,6 @@ pub fn duration(string: &str) -> Result<Duration, String> {
     if let Ok((_left_overs, parsed)) = parsers::parse_duration(string.as_bytes()) {
         Ok(parsed)
     } else {
-        Err(format!("Parser Error: {}", string))
+        Err(format!("Failed to parser Duration: {}", string))
```
Comment (badboy) [inline src/lib.rs:339]:
```suggestion
        Err(format!("Failed to parse duration: {}", string))
```

**Review by badboy — APPROVED** [review]

File: src/lib.rs (hunk 1 above)
Comment (hoodie) [reply src/lib.rs:259]: oh boy, thanks!!!

**Comment by badboy** [comment]
release before merge? that's something. Might want to resolve the conflicts and then still merge it.
Might want to adjust the 0.6.0 tag locally, and then ensure to also push that, once it's in main

**Comment by hoodie** [comment]
oh man, what's going on. I always freaked out by `cargo-release`, it didnt check that I was on the main branch. sorry about this

**Comment by hoodie** [comment]
I just merged this locally

**Comment by hoodie** [comment]
just released 0.6.1 after merging

**Comment by badboy** [comment]
The tag is still not pushed. `git push --tags` in your repo.

**Comment by hoodie** [comment]
will push this tonight. about time git starts pushing tags automatically

**Comment by badboy** [comment]
> will push this tonight. about time git starts pushing tags automatically

```
git config --global push.followTags true
````

;)

**Pull request merged**